e2e/cephfs.go

```diff
@@ -172,549 +194,541 @@ var _ = Describe("cephfs", func() {
+				}
+				// re-define configmap with information of multiple clusters.
+				subvolgrpInfo := map[string]string{
+					"clusterID-1": "subvolgrp1",
+					"clusterID-2": "subvolgrp2",
+				}
+				err = createCustomConfigMap(f.ClientSet, cephfsDirPath, subvolgrpInfo)
+				if err != nil {
+					e2elog.Failf("failed to create configmap %v", err)
+				}
+				err = createCephfsStorageClass(f.ClientSet, f, false, "clusterID-1")
+				if err != nil {
+					e2elog.Failf("failed to create storageclass %v", err)
+				}
+				err = validatePVCAndAppBinding(pvcPath, appPath, f)
+				if err != nil {
+					e2elog.Failf("failed to validate pvc and application %v", err)
+				}
+				err = deleteResource(cephfsExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass %v", err)
+				}
+				// verify subvolumegroup creation.
+				err = validateSubvolumegroup(f, "subvolgrp1")
+				if err != nil {
+					e2elog.Failf("failed to validate subvolume group %v", err)
+				}
 
-					// create resources and verify subvolume group creation
-					// for the second cluster.
-					createCephfsStorageClass(f.ClientSet, f, false, "clusterID-2")
-					validatePVCAndAppBinding(pvcPath, appPath, f)
-					deleteResource(cephfsExamplePath + "storageclass.yaml")
-					err = validateSubvolumegroup(f, "subvolgrp2")
-					if err != nil {
-						Fail(err.Error())
-					}
-					deleteConfigMap(cephfsDirPath)
-				})
+				// create resources and verify subvolume group creation
+				// for the second cluster.
+				err = createCephfsStorageClass(f.ClientSet, f, false, "clusterID-2")
+				if err != nil {
+					e2elog.Failf("failed to create storageclass %v", err)
+				}
+				err = validatePVCAndAppBinding(pvcPath, appPath, f)
+				if err != nil {
+					e2elog.Failf("failed to validate pvc and application %v", err)
+				}
+				err = deleteResource(cephfsExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass %v", err)
+				}
+				err = validateSubvolumegroup(f, "subvolgrp2")
+				if err != nil {
+					e2elog.Failf("failed to validate subvolume group %v", err)
+				}
+				deleteConfigMap(cephfsDirPath)
+				err = createConfigMap(cephfsDirPath, f.ClientSet, f)
+				if err != nil {
+					e2elog.Failf("failed to create configmap %v", err)
+				}
+				err = createCephfsStorageClass(f.ClientSet, f, false, "")
+				if err != nil {
+					e2elog.Failf("failed to create storageclass %v", err)
+				}
+			})
 
-				createConfigMap(cephfsDirPath, f.ClientSet, f)
-				createCephfsStorageClass(f.ClientSet, f, false, "")
+			By("Resize PVC and check application directory size", func() {
+				v, err := f.ClientSet.Discovery().ServerVersion()
+				if err != nil {
+					e2elog.Failf("failed to get server version with error %v", err)
+				}
 
-				By("Resize PVC and check application directory size", func() {
-					v, err := f.ClientSet.Discovery().ServerVersion()
+				// Resize 0.3.0 is only supported from v1.15+
```

Comment:
its good to have a better comment. `Resize 0.3.0 is` ... is confusing.

Reply:
This PR is not adding this log. it was just reorganized